toolkit/tools/imagecreator/main.go

```diff
@@ -28,11 +28,23 @@ type ImageCreatorCmd struct {
 	OutputImageFile   string   `name:"output-image-file" help:"Path to write the customized image to."`
 	OutputImageFormat string   `name:"output-image-format" placeholder:"(vhd|vhd-fixed|vhdx|qcow2|raw)" help:"Format of output image." enum:"${imageformat}" default:""`
 	Distro            string   `name:"distro" help:"Target distribution for the image." enum:"azurelinux,fedora" default:"azurelinux"`
```

Comment:
Let's remove the defaults for `distro` and `distro-version"`.

A default becomes part of the API and therefore is subject to backwards-compatibility restrictions.